### ISSUE TITLE
The training loss curve during the fine-tuning stage

### ISSUE THREAD

**Issue by CnFaker** [wusize/OpenUni#10]
Dear author, thank you for your interesting work. Could you share approximately what the loss was during the fine-tuning stage? I’d like to know whether my training is converging normally. Thank you a lot！
